components/announcement_bar/announcement_bar_controller.tsx

```diff
@@ -19,16 +20,22 @@ type Props = {
     license?: ClientLicense;
     config?: Partial<ClientConfig>;
     canViewSystemErrors: boolean;
+    isCloud: boolean;
```

Comment:
not sure if I understand why are these values needed?

Reply:
Provides the necessary props to the with_get_cloud_subscription component

Reply:
ok, got it. Thanks!